cli/cmd/plugin/unmanaged-cluster/tanzu/tanzu.go

```diff
@@ -135,7 +136,18 @@ func (t *UnmanagedCluster) Deploy(scConfig *config.UnmanagedClusterConfig) (int,
 		log.Style(outputIndent, color.FgYellow).ReplaceLinef("Reading ProviderConfiguration from config file. All other provider specific configs may be ignored.")
 	}
 
-	// 2. Download and Read the TKR
+	// 2. Download and Read the compatible TKR
+
+	// Download compatibility file and read TKr version compatible with version of unmanaged-cluster CLI
+	// when the user did _not_ set the TKr via a flag or configuration option
+	if scConfig.TkrLocation == "" {
+		log.Event(logger.MagnetEmoji, "Resolving and checking TKr compatibility file")
```

Comment:
This will be the **new** first time we mention TKr to the user. Can we spell it out here, and use the abbreviation in subsequent logs?

![image](https://user-images.githubusercontent.com/6200057/159814352-a03bf43c-be1e-4e8b-b4da-d860381f52ce.png)